util/exec_test.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/symflower/eval-dev-quality/log"
+)
+
+func TestCommandWithResult(t *testing.T) {
```

Comment:
Would rename to `TestCommandWithResultTimeout` because it tests one one specific behavior.